docs/quick_tutorial/static_assets.rst

```diff
@@ -43,13 +43,18 @@ Steps
    .. literalinclude:: static_assets/tutorial/static/app.css
     :language: css
 
-#. Make sure we haven't broken any existing code by running the tests:
+#. Add a functional test that requires the newly added static file to be delivered:
+
+   .. literalinclude:: static_assets/tutorial/tests.py
+    :linenos:
```

Comment:
Oh, if you meant for this single instance, yes, please. I answered with the assumption toward wholesale replacement.